docs/workflow/requirements/linux-requirements.md

```diff
@@ -41,15 +47,14 @@ Install the following packages for the toolchain:
 * liblttng-ust-dev
 * libssl-dev
 * libkrb5-dev
-* zlib1g-dev
```

Comment:
The zlib requriement removals is already part of my 3rd part PR: :https://github.com/dotnet/runtime/pull/104399

We're waiting a bit to merge that PR until everything is fully stabilized.

Just mentioning because I see @jkotas suggested you remove it here too.

Reply:
We just need to wait for preview 7 https://github.com/dotnet/runtime/pull/104860#issuecomment-2227093493.

Reply:
Right. What I mean is that I already had these changes in my PR. I don't think we need two PRs doing the same.

Reply:
Ok, I'll revert zlib changes.

Reply:
Ok, @carlossanlop This change is adding a few more zlib references that will need to be deleted in your PR.